src/test/java/com/wikia/webdriver/common/core/url/FandomUrlBuilder.java

```diff
@@ -0,0 +1,49 @@
+package com.wikia.webdriver.common.core.url;
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+
+import org.apache.commons.lang.StringUtils;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.apache.commons.lang.StringUtils'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)